gen/pb-python/sigstore_protobuf_specs/dev/sigstore/rekor/v2/__init__.py

```diff
@@ -67,50 +67,50 @@ class Signature(betterproto.Message):
 
 
 @dataclass(eq=False, repr=False)
-class DsseRequestV002(betterproto.Message):
```

Comment:
@woodruffw any guesses as to why my build output differs from CI? I can just omit this file, I’ve seen this before though.